src/DTO/UserDTO.php

```diff
@@ -125,6 +125,19 @@ class UserDTO
      */
     public $seniority = self::SENIORITY_JUNIOR;
 
+    /**
+     * User location
+     * @var string
+     * @Serializer\Type("string")
+     * @Assert\Choice(choices = {"CHI","NYC","BOS","FRA","PAR","ORL","BUC","BRA","CLU","IAS","HAN","GUA","LYO"},
```

Comment:
Please use Assert/Callback to validate this field. Create a constant in the User class with the list of available locations and use it for validation, documentation & fixtures.